scripts/test-remote-dev-context7-device-login.py

```
@@ -0,0 +1,1024 @@
+        captured["kwargs"] = dict(kwargs)
+        return process
+
+    def fake_killpg(pgid: int, sent_signal: int) -> None:
+        signals.append((pgid, sent_signal))
+
+    try:
+        module.subprocess.Popen = fake_popen
+        module.os.killpg = fake_killpg
+        module.run_login_process(
+            ["synthetic-ctx7"],
+            cwd=Path("/tmp"),
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)